### PR TITLE
feat: add trusted headers variable and setup enhanced getIP

### PR DESCRIPTION
**Pull request by levivannoort**
## What does this PR do?

As not to default on the `x-forwarded-for` header, this pull-request introduces the option to set `trusted headers`. Which are checked from left to right on request presence wise, where the left most address will be used for as the client reference. With a default set to `x-forwarded-for`.

In addition I spotted a missing language reference in `getting started` and the `sdk` references for the `go`.

## Test Plan

Setup tests for the various edge cases I could think of in relation to passing headers into the variable.

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Comment by levivannoort** [comment]
@coderabbitai review

File: src/Appwrite/Utopia/Request.php [hunk 1]
```diff
@@ -233,4 +234,45 @@ public function cacheIdentifier(): string
...
+
+    /**
+     * Get IP
+     *
+     * Returns users IP address based on a list of trusted headers.
+     * Assumes application is only accessible through a trusted proxy.
+     *
+     * @return string
+     */
+    public function getIP(): string
```
Comment (loks0n) [inline src/Appwrite/Utopia/Request.php]: from sync: maybe we can take a array<string> param here instead of depending on env?

reason being that this is extension to library code, should be agnostic to Appwrite-specific env vars

**Review by loks0n — APPROVED** [review]
